src/components/GlobalPreferences/Notifications/SubscriptionsTable.js

```diff
@@ -34,45 +35,16 @@ const SubscriptionsTable = React.memo(function SubscriptionsTable({
     [setSelectedSubscriptions]
   )
 
-  const handleUnsubscribe = useCallback(
-    async e => {
-      setIsSubmitting(true)
-      try {
-        const subscriptionIds = selectedSubscriptions.map(
-          i => subscriptions[i].subscriptionId
-        )
-        await deleteSubscriptions({
-          subscriptionIds,
-          authToken,
-        })
-        // reset selection
-        setIsSubmitting(false)
-        // Refetch subscriptions
-        fetchSubscriptions()
-        setSelectedSubscriptions([])
-      } catch (e) {
-        onApiError(e)
-        setIsSubmitting(false)
-      }
-    },
-    [
-      authToken,
-      fetchSubscriptions,
-      selectedSubscriptions,
-      subscriptions,
-      onApiError,
-    ]
-  )
-
   const organizations = Array.from(
     new Set(subscriptions.map(subscription => subscription.ensName))
   )
   const [selectedOrganization, setSelectedOrganization] = useState(-1)
   const onOrganizationChange = useCallback(
     idx => {
+      setSelectedSubscriptions([])
       setSelectedOrganization(idx)
     },
-    [setSelectedOrganization]
+    [setSelectedOrganization, setSelectedSubscriptions]
```

Comment:
It's mostly cosmetic, but note that the `set*` handlers from `useState()` don't need to be included in dependency lists, since they're guaranteed never to change :).